x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/ConfigInitialNode.java

```diff
@@ -257,15 +255,21 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
             if (nodeKeystore.getSettingNames().contains("xpack.security.transport.ssl.keystore.secure_password") ||
```

Comment:
We should probably bail on any `xpack.security.transport.ssl*` or `xpack.security.http.ssl.*` setting, not just the ones we set, as any would indicate an existing/explicit configuration

Reply:
Yes the code does what you say we should do at the begining when it inspects the yml file.
This check here inspects that we don't overwrite keystone entries.

If the admin supplies keystore settings that don't match the yml settings we shouldn't do(redo) any of the checks that are the responsibility of the nodt to do.

Reply:
> Yes the code does what you say we should do at the begining when it inspects the yml file.

I was talking about secure settings explicitly. But you are right, if there is a PEM file password setting in thekeystore, then there is either a TLS setting in the YML file that would have caused us to bail earlier or it is an invalid configuration that will make the node not start